Contributing.md

```diff
@@ -12,6 +12,7 @@ To get ramped up, please make sure to go through this checklist:
 
 Once your changes are ready for review, please make sure you run through this checklist beforehand:
 
-- [ ] Have you verified your changes  with existing testcases?
-- [ ] Did you update the output `dist/wstrade.js` by running `npm run bundle`?
+- [ ] Have you verified your changes  with existing testcases by running `npm test`?
+- [ ] Did you update the output `wstrade.js` by running `npm run bundle`?
```

Comment:
oh also this is supposed to be `dist.js`, not `wstrade.js`. Maybe i typo'd my previous comment on this.